src/ql/arch/cc/info.cc

```diff
@@ -599,6 +747,14 @@ void Info::populate_backend_passes(pmgr::Manager &manager, const utils::Str &var
 #endif
         }
     );
+    manager.append_pass(
+        "io.cqasm.Report",
+        "scheduledqasmwriter",
+        {
+            {"output_prefix", com::options::global["output_dir"].as_str() + "/%N"},
+            {"output_suffix", "_scheduled.qasm"}
+        }
+    );
```

Comment:
I guess it would be better if there were an API to move existing passes, because then options would be preserved. That doesn't currently exist, but you can also preserve the options manually, rather than just substituting defaults again here. I would do something like:

```c++
if (manager.does_pass_exist("scheduledqasmwriter")) {
    auto options = manager.get_pass("scheduledqasmwriter").get_options();
    manager.remove_pass("scheduledqasmwriter");
    manager.append_pass("io.cqasm.Report", "scheduledqasmwriter").get_options().update_from(options);
}
```

I think that should handle everything properly, without duplicating the option defaults.